internal/inspect/testdata/inspect-full.golden

```diff
@@ -4,10 +4,10 @@ Maintained by: dev <dev@example.com>
 
 some description
 
-Services (2) Replicas Ports     Image
------------- -------- -----     -----
-web1         2        8080-8100 nginx:latest
-web2         2        9080-9100 nginx:latest
+Services (2) Replicas Ports     Original Image Image
+------------ -------- -----     -------------- -----
+web1         2        8080-8100                nginx:latest
```

Comment:
I wonder, if all the column `Original Image` is empty (certainly because the app is not pushed yet), do we need to print this column then?